utils/s3/client.cc

```diff
@@ -16,6 +16,20 @@
 #include "utils/chunked_vector.hh"
 #include "log.hh"
 
+namespace utils {
+
+// FIXME -- the very same thing exists in seastar
```

Comment:
That's fine. Seastar isn't a library for manipulating iovecs.

Reply:
Couldn't resist to make it a one-liner: 

return boost::accumulate(iov | boost::adaptors::transformed(std::mem_fn(&iovec::iov_len)), size_t(0));